falco.yaml

```diff
@@ -907,10 +906,10 @@ log_level: info
 # behavior of the underlying components used by Falco. Only logs of a certain
 # severity level or higher will be emitted. Supported levels: "fatal",
 # "critical", "error", "warning", "notice", "info", "debug", "trace". It is not
-# recommended for production use.
+# recommended to use "debug" and "trace" for production use.
```

Comment:
Out of curiosity, why are these two values recommended for production use ?

From old habits, I would use them when having an issue so for debugging.

Reply:
>  It is **not**
> recommended to use "debug" and "trace" for production use.

:laughing:

Reply:
😅 My bad !